src/plugins/inputs/whois.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+exports.commands = ["whois"];
+
+exports.input = function(network, chan, cmd, args) {
+	// This queries server of the other user and not of the current user, which
+	// does not know idle time.
+	// See http://superuser.com/a/272069/208074.
+	network.irc.raw("WHOIS", args[0], args[0]);
```

Comment:
Good call. How do you suggest I do that? Solve it on the server or just cancel and go back to client-side only?

If server-side, what about:

```js
if (args.length === 1) {
  network.irc.raw("WHOIS", args[0], args[0]);
} else {
  network.irc.raw("WHOIS", args.join(" "));
}
```

or something like that?
`/whois nick` would always be `/whois nick nick`, and everything else would be forwarded as is.